.changeset/dirty-cooks-explode.md

```diff
@@ -0,0 +1,9 @@
+---
+'@astrojs/mdx': major
+---
+
+Handles the breaking change in Astro where content pages (including `.mdx` pages) no longer respond with `charset=utf-8` in the `Content-Type` header.
```

Comment:
```suggestion
Handles the breaking change in Astro where content pages (including `.mdx` pages located within `src/pages/`) no longer respond with `charset=utf-8` in the `Content-Type` header.
```
It's helpful I think to point out that when we say "pages" we mean that the files are treated like page files. With this kind of use being much less common know than Markdown/MDX files in *collections*, unlike before, we typically are pretty explicit when we talk about captial-P page files. 😄